src/Dialog.ts

```diff
@@ -16,29 +16,42 @@
 
 import { assert } from './helper';
 import { CDPSession } from './Connection';
+import Protocol from './protocol';
 
-/* TODO(jacktfranklin): protocol.d.ts defines this
- * so let's ditch this and avoid the duplication
+/**
+ * Dialog instances are dispatched by `Page` via the `dialog` event.
```

Comment:
once we have documented `Page` and the events we can turn these into links to those docs.